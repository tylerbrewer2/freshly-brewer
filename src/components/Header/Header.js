import React from 'react';
import styled from 'styled-components';

const HeaderCore = ({ className, tag, children }) => {
  const HeaderTag = tag;

  return (
    <HeaderTag className={className}>{children}</HeaderTag>
  )
}

const Header = styled(HeaderCore)`
  margin: 0;
  font-family: "FreightBook", "serif";
  letter-spacing: 3px;  
`

export default Header;