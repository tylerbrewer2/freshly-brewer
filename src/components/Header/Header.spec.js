import React from 'react';
import renderer from 'react-test-renderer';

import Header from './Header';

it('renders correctly', () => {
  const tree = renderer.create(
    <Header tag="h4">
      Test Header
    </Header>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
