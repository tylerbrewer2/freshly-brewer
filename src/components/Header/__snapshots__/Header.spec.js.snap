exports[`test renders correctly 1`] = `
<h4
  className="sc-bdVaJa bErzIe">
  Test Header
</h4>
`;
